vm/src/builtins/property.rs

```diff
@@ -217,6 +217,32 @@ impl PyProperty {
         }
         .into_ref_with_type(vm, TypeProtocol::clone_class(&zelf))
     }
+
+    #[pyproperty(magic)]
+    fn isabstractmethod(&self, vm: &VirtualMachine) -> PyObjectRef {
+        let getter_abstract = match self.getter.read().to_owned() {
```

Comment:
@coolreader18 is `to_owned` the best way to handle this?

Reply:
@coolreader18 do you have any idea about this? or i will merge this PR

Reply:
@youknowone & @coolreader18, Just wanted to check in on this PR. Should I work on trying to remove the `to_owned()`?